### PR TITLE
Fluent 2 Reset: Update design-tokens

### PR DESCRIPTION
**Pull request by levithomason**
This PR adds/updates design tokens to match Fluent 2.

- Added some comments to help when quick find things when scrolling
- `borderRadius`: made some additions/fixes
- `shadow`: added references to `ambient` and `key` shadow colors (not sure if syntax is idiomatic)

### CODE REVIEW

File: packages/web-components/src/design-tokens.ts [hunk 1]
```diff
@@ -2,10 +2,14 @@ import { DesignToken } from '@microsoft/fast-foundation';
...
 
+// ----------------------------------------
+// Border Radius
+// ----------------------------------------
+
 /**
  * @public
  */
-export const borderRadiusNone = create<number>('borderRadiusNone').withDefault(0);
+export const borderRadiusNone = create<string>('borderRadiusNone').withDefault('0');
```
Comment (chrisdholt) [inline packages/web-components/src/design-tokens.ts:12]: Thanks for this - I was wondering how to best handle this as it's an odd case to override a token for "none" with anything but zero, but it makes sense.

File: packages/web-components/src/design-tokens.ts [hunk 2]
```diff
@@ -2003,78 +2051,94 @@ export const colorPaletteAnchorForeground2 = create<string>('colorPaletteAnchorF
...
+// ----------------------------------------
+// Shadow
+// ----------------------------------------
+
 /**
  * @public
  */
-export const shadow2 = create<string>('shadow2').withDefault('0 0 2px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.14)');
+export const shadow2 = create<string>('shadow2').withDefault(
+  `0 0 2px ${colorNeutralShadowAmbient.$value}, 0 1px 2px ${colorNeutralShadowKey.$value}`,
```
Comment (chrisdholt) [inline packages/web-components/src/design-tokens.ts]: I'm not sure you need `$value` here, you should be able to reference the token itself.

**Review by chrisdholt — APPROVED** [review]
Looks good for the most part - thanks for catching the radius issues, I just fixed that locally when working on checkbox :).

One note on the invocation of `$value` as I think referencing the token should work here.

File: packages/web-components/src/design-tokens.ts (hunk 2 above)
Comment (levithomason) [reply packages/web-components/src/design-tokens.ts]: Will do. Does the `$` prefix mean private, or something else?

Comment (chrisdholt) [reply packages/web-components/src/design-tokens.ts]: No but in the use you have you're essentially just interpolating the value of another token so my thought is it should just be a direct reference...

**Pull request merged**